frontend/libs/studio-components/README.md

```diff
@@ -0,0 +1,13 @@
+# @altinn/studio-components
+
+Studio-Components is a tools that wraps around `@digdir/design-system-react` and `@digdir/design-system-tokens`, giving you access to the Designsystem's componetns and tokens adding Altinn customisation to them. It's great for delvelopers and designers to have this library for components to ensure that the components used throughout altinn studio's applications looks great and stay consistent.
+
+## Contribute with new components
```

Comment:
Jeg stemmer for å bygge komponentene på samme måte som vi gjør i digdir-designsystemet. Som bildet under viser, så utvider vi typen med `HTMLAttributes `og typen til elementet som brukes på toppnivå. Dette gjør at vi automatisk kan sende med alle props som tilhører native elementet. Samtidig så bruker vi forwardRef for å sørge for å støtte React Top Level API. 😊 


![image](https://github.com/Altinn/altinn-studio/assets/46874830/3484d452-a834-41d5-9d00-d6b977fc7dd4)

Reply:
Bildet er tatt ut fra en medium artikkel jeg skrev etter jeg hadde vært en stund på designsystemet. https://medium.com/bitsrc/what-makes-great-react-components-great-209876d0c8de. Jeg synes vi skal legge inn litt innsats for å bygge slike komponenter. 

Ved å gjøre det på denne måten så vil vi lettere kunne følger prinsipper som "closed for modification, but open for extension". Det er også en rekke andre fordeler med å følge denne måten å bygge komponenter på. 😊

Reply:
Ja, 100% enig 😄